RF24.cpp

```diff
@@ -783,6 +817,175 @@ void RF24::printPrettyDetails(void) {
+                                     PRIPSTR
+                                     "\n"
+                                     "CRC Length\t\t"
+                                     PRIPSTR
+                                     "\n"
+                                     "Address Length\t\t= %d bytes\n"
+                                     "Static Payload Length\t= %d bytes\n"
+                                     "Auto Retry Delay\t= %d microseconds\n"
+                                     "Auto Retry Attempts\t= %d maximum\n"
+                                     "Packets lost on\n    current channel\t= %d\r\n"
+                                     "Retry attempts made for\n    last transmission\t= %d\r\n"
+                                     "Multicast\t\t"
+                                     PRIPSTR
+                                     "\n"
+                                     "Custom ACK Payload\t"
+                                     PRIPSTR
+                                     "\n"
+                                     "Dynamic Payloads\t"
+                                     PRIPSTR
+                                     "\n"
+                                     "Auto Acknowledgment\t"
+                                     PRIPSTR
+                                     "\n"
+                                     "Primary Mode\t\t= %cX\n"
+                                     "TX address\t\t= 0x"
+                                     PRIPSTR
+                                     "\n"
+                                     "pipe 0 ( "
+                                     PRIPSTR
+                                     " ) bound\t= 0x"
+                                     PRIPSTR
+                                     "\n"
+                                     "pipe 1 ( "
+                                     PRIPSTR
+                                     " ) bound\t= 0x"
+                                     PRIPSTR
+                                     "\n"
+                                     "pipe 2 ( "
+                                     PRIPSTR
+                                     " ) bound\t= 0x"
+                                     PRIPSTR
+                                     "\n"
+                                     "pipe 3 ( "
+                                     PRIPSTR
+                                     " ) bound\t= 0x"
+                                     PRIPSTR
+                                     "\n"
+                                     "pipe 4 ( "
+                                     PRIPSTR
+                                     " ) bound\t= 0x"
+                                     PRIPSTR
+                                     "\n"
+                                     "pipe 5 ( "
+                                     PRIPSTR
+                                     " ) bound\t= 0x"
+                                     PRIPSTR
+                                     "\n\n"
```

Comment:
I don't think this extra `\n\n` is really necessary as the user can control how the last line terminates.

Reply:
we could probably also save an extra 12 bytes by getting rid of the padded spaces around the `closed`/` open ` status for each pipe.
Changing
```
pipe 0 (  open  ) bound = 0x65646f4e32
pipe 1 (  open  ) bound = 0x65646f4e31
pipe 2 ( closed ) bound = 0xc3
pipe 3 ( closed ) bound = 0xc4
pipe 4 ( closed ) bound = 0xc5
pipe 5 ( closed ) bound = 0xc6
```
into 
```
pipe 0 ( open ) bound = 0x65646f4e32
pipe 1 ( open ) bound = 0x65646f4e31
pipe 2 (closed) bound = 0xc3
pipe 3 (closed) bound = 0xc4
pipe 4 (closed) bound = 0xc5
pipe 5 (closed) bound = 0xc6
```

Reply:
With the above changes, I calculate a maximum 861 bytes needed for any resulting output from `sprintfPrettyDetails()` This calculation includes
- binary representation of auto-ack
- LNA = "Disabled"
- Channel > 99
- max values of "Packets lost ..." & "Retry attempts made ..."

Reply:
@dstroy0 If your scenario is ok with these changes, then I'll commit them. I'm not sure if the `\n\n` is required for sending the c-string over MQTT, but I think it can still be added on the user's end not the lib's end.